schema/site_metadata.json

```diff
@@ -50,6 +58,42 @@
             "type": "number"
         }
       }
+    },
+    "area": {
+      "type": "object",
+      "additionalProperties": false,
+      "properties": {
+        "m2": {
+            "description": "Area of the site in square meters",
+            "type": "number",
+            "examples": [1234, 256, 998765]
+        },
+        "sqf": {
+            "description": "Area of the site in square feet",
+            "type": "number",
+            "examples": [13282.67, 2755.56, 10750617]
+        }
+      }
+    },
+    "site_folder": {
```

Comment:
maybe bucket these into a `source` object?  There's also other things that come up, e.g. the branch, folder, potentially a specific commit. And eventually there might be other mechanisms (e.g. that's not a git repo), so then it'd be a place to contain a type-of-source enum, etc... ?

```
source.folder
source.repo
source.????
```

Reply:
LGTM, I'll update it.

Reply:
Done